### PR TITLE
Feat: added update-spreadsheet action

### PR DESCRIPTION
**Pull request by GarvitSinghal47**
## Summary

…

## References

…

## Reviewer guidance

…

### CODE REVIEW

**Comment by MisRob** [comment]
@rtibbles in this repo, would you please enable ` GH_UPLOADER_GCP_SA_CREDENTIALS` as well? Sheet secrets are already here. Thank you.

**Review by MisRob — APPROVED** [review]
All looks good to me here! Thank you @GarvitSinghal47. I will let @rtibbles to make final confirmation before I merge.

Also @rtibbles in this repo, would you please enable ` GH_UPLOADER_GCP_SA_CREDENTIALS` as well? All other sheet secrets are already here. Thank you.

File: .github/workflows/update-pr-spreadsheet.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: PR Event Workflow
```
Comment (MisRob) [inline .github/workflows/update-pr-spreadsheet.yml:1]: @GarvitSinghal47 Just a little detail, nothing blocking. if we could rename to "Update contributions spreadsheet" on this other related PRs, it'd help us when previewing all action runs. I can do it later as well.

Comment (MisRob) [inline .github/workflows/update-pr-spreadsheet.yml:1]:
```suggestion
name: Update community pull requests spreadsheet
```

**Review by rtibbles — APPROVED** [review]

**Pull request merged**